src/controllers/breaches.js

```diff
@@ -56,19 +56,24 @@ async function putBreachResolution (req, res) {
   const sessionUser = req.user
   const { affectedEmail, breachId, resolutionsChecked } = req.body
   const breachIdNumber = Number(breachId)
-  const affectedEmailIsSubscriberRecord = sessionUser.primary_email === affectedEmail
-  const affectedEmailInEmailAddresses = sessionUser.email_addresses.filter(ea => ea.email === affectedEmail)
+  const affectedEmailAsSubscriber = sessionUser.primary_email === affectedEmail ? sessionUser.primary_email : false
+  const affectedEmailInEmailAddresses = sessionUser.email_addresses.filter(ea => ea.email === affectedEmail)?.[0]?.email || false
```

Comment:
Would `Array.find()` be more appropriate here?
```suggestion
  const affectedEmailInEmailAddresses = sessionUser.email_addresses.find(ea => ea.email === affectedEmail)?.email || false
```

Reply:
Awesome